CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/screens/home/HomeViewModel.kt

```diff
@@ -49,10 +50,10 @@ class HomeViewModel @Inject constructor(
 
     private fun fetchUsers() {
         showLoading()
-        viewModelScope.launch {
-            when (val result = getUsersUseCase.execute()) {
-                is UseCaseResult.Success -> _users.value = result.data
-                is UseCaseResult.Error -> _error.emit(result.exception.message.orEmpty())
+        execute {
+            when (val result = getUsersUseCase.execute(page = 1, size = 10)) {
+                is co.nimblehq.domain.usecase.UseCaseResult.Success -> _users.value = result.data
```

Comment:
Let's import `import co.nimblehq.domain.usecase.UseCaseResult` so we can use just `UseCaseResult.Success` 🙏

Reply:
@Wadeewee Sure, I've just copied and pasted from previous version without changing.